app/controllers/deployments_controller.rb

```diff
@@ -79,6 +107,10 @@ def push_notification?
     params[:repo].present?
   end
 
+  def use_application_by_name
+    params[:application_by_name].present? && params[:application_by_name] == "true"
```

Comment:
`params.fetch(:application_by_name, nil) == "true"` would be another way to write this I think, where `nil` is the default value if the key isn't present.

Reply:
I will do that.